portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/languageportlet/LanguagePortlet.macro

```diff
@@ -91,6 +91,13 @@ definition {
 			locator1 = "Language#DROPDOWN_CURRENT_LANGUAGE");
 	}
 
+	macro viewImportFileUploadBorderBottomColor {
```

Comment:
Instead of a ew macro, try to find the exist macro to confirm the field is a required one. Thanks.